docs/app-development.rst

```diff
@@ -215,6 +225,22 @@ The block header will be updated (TODO) to include some commitment to
 the results of DeliverTx, be it a bitarray of non-OK transactions, or a
 merkle root of the data returned by the DeliverTx requests, or both.
 
+::
+
+    // tx is either "key=value" or just arbitrary bytes
```

Comment:
we need to fix this comment because tx can also be `val:pubkey/power`

Reply:
fixed in abci:develop